README.md

```diff
@@ -1,6 +1,6 @@
-# Work in progress ⚠
+# In review ⚠
 
--   **The library is not ready.** We aim at a beta release in the following months. If interested in a demo/early beta, please reach out to us.
+-   **The library is currently under review.** We expect there to be changes as the community provides feedback. If interested in a demo/early beta, please reach out to us.
```

Comment:
I would remove "If interested in a demo/early beta, please reach out to us.", otherwise good to go!

Reply:
same!